src/check.ts

```diff
@@ -1,13 +1,26 @@
-import CreditCard from 'card-validator';
-import { userSession } from './js/sessiondata';
+/* eslint-disable complexity */
 import { modalWindows } from './js/modals';
+import idpayguard from './js/idpayguard';
+import { initHeader } from './js/header';
+import { setTranslateBtns } from './js/translateui';
 
-document.addEventListener('DOMContentLoaded', function () {
-  userSession();
-
+// eslint-disable-next-line sonarjs/cognitive-complexity
+document.addEventListener('DOMContentLoaded', () => {
+  // idpayguard
+  idpayguard();
+  // initHeader
+  initHeader();
+  // init translations
+  setTranslateBtns();
   // init modals
   modalWindows();
 
+  const walletStored = sessionStorage.getItem('wallet') || '';
+  const checkDataStored = sessionStorage.getItem('checkData') || '';
+  const wallet = JSON.parse(walletStored);
+  const checkData = JSON.parse(checkDataStored);
+  const userEmail = sessionStorage.getItem('useremail') || '';
```

Comment:
Why doesn't retrieve all JSON data in one line ? Is there a reason ? 
Doing so, you put immediately attention to the variables used later on